speakers.js

```diff
@@ -0,0 +1,50 @@
+function Speaker(name, profession, description, featureImg) {
+  this.name = name;
+  this.profession = profession;
+  this.description = description;
+  this.featureImg = featureImg;
+}
+
+const speakers = [];
```

Comment:
Increase the speaker number as shown in the original design below
#### Original design
![right1](https://user-images.githubusercontent.com/58376020/125110939-9103a580-e0e5-11eb-9d95-799882e72a11.png)

#### Your Project
![wrong1](https://user-images.githubusercontent.com/58376020/125111025-a973c000-e0e5-11eb-965f-69f86190fe0a.png)